### PR TITLE
Add API to resolve mount id from ufs path in FileSystem

### PR DESCRIPTION
**Pull request by yyongycy**
Add API to resolve mount id from ufs path in FileSystem

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java [hunk 1]
```diff
@@ -112,6 +112,14 @@ FileInfo getFileInfo(AlluxioURI path, GetStatusContext context)
       throws FileDoesNotExistException, InvalidPathException, AccessControlException,
       UnavailableException, IOException;
 
+  /**
+   * Returns the mount id according to the ufs path.
+   *
+   * @param ufsPath the ufs path
+   * @return the corresponding mount it
```
Comment (LuQQiu) [inline core/server/master/src/main/java/alluxio/master/file/FileSystemMaster.java:119]: mount it -> mount id

**Review by LuQQiu — APPROVED** [review]
LGTM after fixing the typo

**Comment by yyongycy** [comment]
@LuQQiu  fixed the typo

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**